dagger-common/src/main/java/io/odpf/dagger/common/serde/typehandler/repeated/RepeatedEnumHandler.java

```diff
@@ -54,7 +55,19 @@ public Object transformFromProto(Object field) {
 
     @Override
     public Object transformFromParquet(SimpleGroup simpleGroup) {
-        return null;
+        String defaultEnumValue = fieldDescriptor.getEnumType().findValueByNumber(0).getName();
+        List<String> deserializedEnumArray = new ArrayList<>();
```

Comment:
enumArray? For consistency with other similar classes?

Reply:
will do this

Reply:
Fixed via commit https://github.com/odpf/dagger/pull/148/commits/a28474cef99f9a873be2ff405380bd27f2a9f0b7